src/components/product/Pagination.module.css

```diff
@@ -0,0 +1,59 @@
+.paginationWrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 30px;
+  margin-top: 60px;
+}
+
+.arrowButton {
+  width: 32px;
+  height: 32px;
+  box-shadow: 0 0 0 1px var(--pagehover);
+  color: var(--text-maincolor);
+  border-radius: 8px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+}
+
+.arrowButton:hover {
```

Comment:
Please correct the color of disabled `>` - should be `#ccc`
Also would be good to block any interaction with disabled button - not change bg-color on hover and change the cursor to default
![image](https://github.com/demasha0908/fe-react-2024/assets/98031768/bd0dcc24-c36f-4929-9bb7-5136b215ed4d)